static/src/inline-svgs/icon/twitter-bird.svg

```diff
@@ -1,3 +1 @@
-<svg width="24px" height="24px" viewBox="0 0 24 24">
-    <path d="M18.999,7.346 C18.481,7.574 17.931,7.729 17.349,7.797 C17.942,7.441 18.397,6.879 18.614,6.209 C18.058,6.537 17.442,6.775 16.79,6.906 C16.269,6.346 15.519,6 14.692,6 C13.106,6 11.819,7.285 11.819,8.871 C11.819,9.098 11.845,9.314 11.895,9.523 C9.508,9.406 7.394,8.261 5.977,6.523 C5.729,6.949 5.587,7.441 5.587,7.967 C5.587,8.962 6.095,9.842 6.866,10.357 C6.395,10.343 5.952,10.212 5.564,10 L5.564,10.035 C5.564,11.428 6.554,12.588 7.868,12.85 C7.627,12.915 7.373,12.953 7.111,12.953 C6.926,12.953 6.745,12.934 6.57,12.898 C6.936,14.043 7.997,14.873 9.253,14.894 C8.27,15.666 7.03,16.125 5.685,16.125 C5.454,16.125 5.226,16.109 5,16.086 C6.272,16.898 7.781,17.375 9.403,17.375 C14.687,17.375 17.574,13 17.574,9.205 C17.574,9.08 17.572,8.955 17.564,8.832 C18.126,8.426 18.614,7.92 18.999,7.346"></path>
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" width="14" height="12" viewBox="0 0 14 12"><path d="M-274 392.3c-.5.2-1.1.4-1.6.5.6-.4 1-.9 1.3-1.6-.6.3-1.2.6-1.8.7-.5-.6-1.3-.9-2.1-.9-1.6 0-2.9 1.3-2.9 2.9 0 .2 0 .4.1.7-2.4-.1-4.5-1.3-5.9-3-.2.4-.4.9-.4 1.4 0 1 .5 1.9 1.3 2.4-.5 0-.9-.1-1.3-.4 0 1.4 1 2.6 2.3 2.8-.2.1-.5.1-.8.1-.2 0-.4 0-.5-.1.4 1.1 1.4 2 2.7 2-1 .8-2.2 1.2-3.6 1.2h-.7c1.3.8 2.8 1.3 4.4 1.3 5.3 0 8.2-4.4 8.2-8.2v-.4c.4-.3.9-.8 1.3-1.4"/></svg>

```

Comment:
@Jholder112233 It might be that we don't add icons that frequently but is there an opportunity to automate the removal of `xmlns` given someone mentions this every time an icon is added or updated?